templates/about.pug

```diff
@@ -1,10 +1,6 @@
-doctype html
-html(lang= "en")
-  head
-    title about
+extends index.pug
```

Comment:
You might want to call it "layout" I think? `index` sounds more like a page rather than a layout to me ;)